demos/python_demos/llm_text_generation/README.md

```diff
@@ -50,46 +106,78 @@ Depending on the use case, `./servable_unary` and `./servable_stream` showcase d
 
 To test unary example:
 ```bash
-docker run -it --rm -p 9000:9000 -v ${PWD}/servable_unary:/workspace -v ${PWD}/model:/model openvino/model_server:py --config_path /workspace/config.json --port 9000
+docker run -d --rm -p 9000:9000 -v ${PWD}/servable_unary:/workspace -v ${PWD}/${SELECTED_MODEL}:/model \
+-e SELECTED_MODEL=${SELECTED_MODEL} openvino/model_server:py --config_path /workspace/config.json --port 9000
 ```
 
-## Requesting the LLM
+You can also deploy the compressed model by just changing the model path mounted to the container. For example:
+
+```bash
+docker run -d --rm -p 9000:9000 -v ${PWD}/servable_unary:/workspace -v ${PWD}/${SELECTED_MODEL}_INT8_compressed_weights:/model \
+-e SELECTED_MODEL=${SELECTED_MODEL} openvino/model_server:py  --config_path /workspace/config.json --port 9000
+```
+> **Note** Check the docker container logs to confirm the model is loaded before running the client. Depending on the model and hardware it might take up to several minutes.
+
+### Running the client with LLM model and unary gRPC call
+
+Install python client dependencies. This is a common step also for the streaming client.
+```bash
+pip install -r client_requirements.txt
+```
 
 Run time unary client `client_unary.py`:
 ```bash
-python3 client_unary.py --url localhost:9000 --prompt "Describe the state of the healthcare industry in the United States in max 2 sentences"
+python3 client_unary.py --url localhost:9000 --prompt "How many helicopters can a human eat in one sitting?"
```

Comment:
@dtrawins  can we change this prompt to something more realistic? I suggest changing helicopters to apples or coming up with a new prompt.